streampark-common/src/main/scala/org/apache/streampark/common/conf/K8sFlinkConfig.scala

```diff
@@ -18,37 +18,54 @@
 package org.apache.streampark.common.conf
 
 /** Flink kubernetes Configuration for v1 version */
-@deprecated("see: org.apache.streampark.flink.kubernetes.v2.Config")
+
 object K8sFlinkConfig {
 
+  lazy val isV2Enabled: Boolean = InternalConfigHolder.get(ENABLE_V2)
+
+  val ENABLE_V2: InternalOption = InternalOption(
+    key = "streampark.flink-k8s.enable-v2",
+    defaultValue = false,
+    classType = classOf[Boolean],
+    description =
+      "Whether to enable the v2 version(base on flink-kubernetes-operator) of flink kubernetes operation"
+  )
+
+  //  ======= deprecated =======
+
+  @deprecated
```

Comment:
StreamPark consists of two parts: a development framework and a stream processing platform. The development framework part provides some APIs to users. The parameter InternalOption was originally designed for internal use within the platform, It is a non-public API. Therefore, these parameters can be deleted directly without causing any other impact on users